stock_by_warehouse/static/src/js/widget.js

```diff
@@ -80,7 +80,8 @@ var ShowPaymentLineWidget = form_common.extend({
                 'placement': this.nodeOptions.placement || 'right',
                 'title': this.nodeOptions.title || self.info.title || "",
                 'trigger': 'focus',
-                'delay': { "show": 0, "hide": 100 }
+                'delay': { "show": 0, "hide": 100 },
+                'template': '<div class="popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body stock-by-warehouse-widget"></div></div>'
```

Comment:
I think this is too complex to be harcoded as a string, you could use `qweb.render("template_name")` and move this to a template

Reply:
as I understood and following the @imanie383 recommendations, this is the best way, at least enough to solve the issue, or what do you think @imanie383?

Reply:
@luisg123v  is right